routes/donations.js

```diff
@@ -108,8 +173,9 @@ router.put('/:donationId', async (req, res) => {
       email,
       phoneNum,
       notes,
-      date,
+      submittedDate,
```

Comment:
`submittedDate` won't be an argument passed into this route, you can go ahead and delete it